tls/s2n_cipher_suites.c

```diff
@@ -680,6 +680,21 @@ struct s2n_cipher_suite s2n_dhe_rsa_with_chacha20_poly1305_sha256 = /* 0xCC,0xAA
 };
 
 /* From https://tools.ietf.org/html/draft-campagna-tls-bike-sike-hybrid-01 */
```

Comment:
Suggest removing the `-01` at the end of the URL so that the link goes directly to the most recent revision (Kyber isn't included in revision 01).